docs/development/extensions-core/druid-lookups.md

```diff
@@ -33,6 +33,12 @@ This module can be used side to side with other lookup module like the global ca
 
 To use this extension please make sure to  [include](../../development/extensions.md#loading-extensions) `druid-lookups-cached-single` as an extension.
 
+> If using JDBC, you will need to add your database's client JAR files to the extension's directory.
```

Comment:
Can you add a similar note to `lookups-cached-global.md`?

Reply:
A note was previously added for global lookups by #8825 :
https://github.com/apache/incubator-druid/pull/8825/files#diff-2eebc164c4539ea1db720a542e33db76R371-R376